cicecore/cicedyn/dynamics/ice_dyn_eap.F90

```diff
@@ -1357,6 +1366,13 @@ subroutine stress_eap  (nx_block,   ny_block,       &
                  (tensionne + tensionnw + tensionse + tensionsw)**2 &
                 +  (shearne +   shearnw +   shearse +   shearsw)**2)
 
+            ! vorticity
+            dvdxn = dyU(i,j)*vvel(i,j) - dyU(i-1,j)*vvel(i-1,j)
+            dvdxs = dyU(i,j-1)*vvel(i,j-1) - dyU(i-1,j-1)*vvel(i-1,j-1)
+            dudye = dxU(i,j)*uvel(i,j) - dxU(i,j-1)*uvel(i,j-1)
+            dudyw = dxU(i-1,j)*uvel(i-1,j) - dxU(i-1,j-1)*uvel(i-1,j-1)
+            vort(i,j) = p5*tarear(i,j)*(dvdxn + dvdxs - dudye - dudyw)
+
```

Comment:
Here the computation of vorticity is the same as in `ice_dyn_shared::deformations`, right ?

If so, I would prefer we compute the vorticity in a separate subroutine, and call that subroutine from `ice_dyn_eap::stress_eap` and  `ice_dyn_shared::deformations`. This would reduce code duplication.

Reply:
But then it is the same thing for divu and shear...the same computations are done in ice_dyn_eap and ice_dyn_shared. It has been like this for a long time...I say we keep it simple the way it is done right now.